### ISSUE TITLE
Client crash after restarting the resource

### ISSUE THREAD

**Issue by mrgharabaghi** [altmp/altv-issues#1531]
### Description of the problem

If you restart one of your resources and then press `Esc`, the client will crash.
This happens with `debug = false` and `debug = true`.

### Reproduction steps

1. Start the server with a normal `bat` file with `altv-server.exe` in it.
2. Restart `res` with `restart res`.
3. Press `Esc` key.
4. The client will crash.

`server.cfg`
```
resources: [
    'res'
]
```

### Expected behaviour

It shouldn't be crashed.

### Additional context

`crashdump`
- [additional.txt](https://github.com/altmp/altv-issues/files/9210153/additional.txt)
- [client.log](https://github.com/altmp/altv-issues/files/9210163/client.log)

### Operating system

Windows 10

### Version

release/12.5

**Comment by FabianTerhorst** [comment]
Ask for Help on discord.

**Comment by YannBcf** [comment]
@mrgharabaghi Can you try on dev branch, just tried 3 times there and I couldn't reproduce

**Comment by deluvas1911** [comment]
do you use any native like e.g. skycam? because the orange screen in your video seems like you do. could be also related to tabbing out while a certain native is processing

**Comment by mrgharabaghi** [comment]
If this crash happens in `dev/11`, I will send a new comment.

**Comment by mrgharabaghi** [comment]
> do you use any native like e.g. skycam?

No

**Comment by mrgharabaghi** [comment]
If you restart one of your resources and then press `Esc`, the client will crash.
This happens with `debug = false` and `debug = true`.

1. Start the server with a normal `bat` file with `altv-server.exe` in it.
2. Restart `res` with `restart res`.
3. Press `Esc` key.
4. The client will crash.

`server.cfg`
```
resources: [
    'res'
]
```
`crashdump`
- [additional.txt](https://github.com/altmp/altv-issues/files/9210153/additional.txt)
- [client.log](https://github.com/altmp/altv-issues/files/9210163/client.log)

`v12.5 release`

**Comment by deluvas1911** [comment]
Can confirm with the behaviour described in the above comment
@mrgharabaghi can you the text of this comment to the issues initial post? so it will be easier for others to directly find the right reproduction

**Comment by C0kkie** [comment]
Fixed in upcoming release release/12.9

**Comment by mrgharabaghi** [comment]
I've got this error only for `subdirectories` after restarting the resource on the client side.
`restart map/ipl`
```
[23:56:35][Error] HTTP Request failed, curl code: 22
[23:56:35][Error] Connection failed: Failed to download resource map/ipl
[23:56:35][Error] Failed to download resource: RESOURCE_DOWNLOAD_FAILED
```

**Comment by mrgharabaghi** [comment]
> I've got this error only for `subdirectories` after restarting the resource on the client side. `restart map/ipl`
> 
> ```
> [23:56:35][Error] HTTP Request failed, curl code: 22
> [23:56:35][Error] Connection failed: Failed to download resource map/ipl
> [23:56:35][Error] Failed to download resource: RESOURCE_DOWNLOAD_FAILED
> ```

Fixed in release/12.10
